docker/00-add_local_path.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Commit message: Again, please don't emphasize "multi stage". It has been multi-stage before.